Firmware/cmdqueue.cpp

```diff
@@ -478,6 +478,9 @@ void get_command()
         // Handle KILL early, even when Stopped
         if(strcmp(cmdbuffer+bufindw+CMDHDRSIZE, "M112") == 0)
           kill(MSG_M112_KILL, 2);
+        // Stop print
+        if (strcmp(cmdbuffer+bufindw+CMDHDRSIZE, "M603") == 0)
```

Comment:
Please change this to strcmp_P and wrap the string with PSTR(). Do this also to the M112 above and maybe also change this to a `else if` since it might save a few bytes of flash